keyboards/yetis/info.json

```diff
@@ -0,0 +1,99 @@
+{
+    "manufacturer": "jogme",
+    "keyboard_name": "Yetis",
+    "maintainer": "qmk",
+    "bootloader": "rp2040",
```

Comment:
```suggestion
    "development_board": "elite_pi",
```

Reply:
done, but can't remove the bootloader

Reply:
Actually if the board is intended to work with any kind of Elite-C compatible controller, this should be `"development_board": "elite_c"` (and then the rest of code should be configured with the ATMega32U4 pin names and such).  Unfortunately, it's not possible to use anything other than `promicro` or `elite_c` as the original controller for use with converters.

Do you intend to support the usage of any other controllers than `rp2040_ce`-compatible ones with this board?

Reply:
Yes, the board is designed to work with `elite-c` compatible controller. The one I'm currently using (Frood, but see f.e. helios, elite-pi) has an elite-c pin layout with RP2040 under the hood.

I don't intend to support pi pico pin layout

Reply:
I meant either the original Elite-C with ATmega32U4, or maybe the [STeMCell](https://megamind4089.github.io/STeMCell/) controller (STM32F411) — apparently there are no RP2040-based controllers with Elite-C pins which are not compatible with the `rp2040_ce` pinout, so the only different options at the moment are those two.

Reply:
Since it seems like you used non-standard pins for the OLED (GP0, GP1), I don't think `development_board` is the best bet... something like this is required
`info.json`
```json
"processor": "RP2040",
"bootloader": "rp2040"
```
`config.h`
```c
// Copyright 2023 QMK
// SPDX-License-Identifier: GPL-2.0-or-later
#pragma once

#define I2C_DRIVER I2CD0
#define I2C0_SDA_PIN GP0
#define I2C0_SCL_PIN GP1
```
`halconf.h`
```c
// Copyright 2023 QMK
// SPDX-License-Identifier: GPL-2.0-or-later
#pragma once

#define HAL_USE_I2C TRUE

#include_next <halconf.h>
```
`mcuconf.h`
```c
// Copyright 2023 QMK
// SPDX-License-Identifier: GPL-2.0-or-later
#pragma once
#include_next <mcuconf.h>

#undef RP_I2C_USE_I2C0
#define RP_I2C_USE_I2C0 TRUE
```

Reply:
> Since it seems like you used non-standard pins for the OLED (GP0, GP1)

This actually looks like some confusion — those defines are specific to the SPI OLED interface, and actually don't do anything for an I2C OLED:
```c
#define OLED_DC_PIN GP1
#define OLED_CS_PIN GP0
```
The actual PCB as published at https://github.com/jogme/yetis-keyboard looks like it uses the standard I2C pin locations for Elite-C (and it does not use the problematic B0/RAW pin), so it could potentially be compatible with other controllers.

Reply:
I decided to go with the 'elite_c' with conversion to 'rp2040_ce'. Hope that's alrigh